app/code/Magento/CustomerGraphQl/Model/Customer/UpdateCustomerAccount.php

```diff
@@ -105,4 +113,29 @@ public function execute(CustomerInterface $customer, array $data): void
             $this->changeSubscriptionStatus->execute((int)$customer->getId(), (bool)$data['is_subscribed']);
         }
     }
+
+    /**
+     * @param array $customerData
+     * @return void
+     * @throws GraphQlInputException
+     */
+    public function validateData(array $customerData): void
```

Comment:
Could be reuse logic which is used in frontend (or REST)?
Because we will have 2 way of validation: on Magento frontend and validation in GraphQL

Reply:
Done